omg/entities/player.py

```diff
@@ -51,26 +55,43 @@ def __init__(self, name, char_class, image_file, scale, initial_angle=0):
         self.mana_regen_cooldown = 0
 
         # Skills
-        self.skills = SkillManager(N_SKILLS_MAX)
+        self.element = SkillManager(N_SKILLS_MAX)
```

Comment:
Discussion point: You are now directing the game to be wizard-based. But I think we can keep the namings general which would allow it to make sense with warriors / rogues etc. `Skill` in my opinion was a generic enough name but the `Element` might not be, unless we define warrior/rogue skills based on elements.